PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Please try to adhere to the guidelines below as much as possible when submitting your pull request.
+
+- Please verify that your code satisfies the [code/documentation quality guidelines](http://scikit-learn.org/stable/developers/contributing.html#coding-guidelines).
+- Please prefix the title of your pull request with `[MRG]` if the contribution is complete and should be subjected to a detailed review. Otherwise:
```

Comment:
I feel like its good to contrast the two options that users might have at this stage.